catlearn/relation_cache.py

```diff
@@ -15,6 +16,67 @@
 BinaryOp = Callable[[Tsor, Tsor], Tsor]
 
 
+def kl_match(
+        score: Tsor, label: Optional[Tsor] = None,
+        against_negative: bool = False) -> Tsor:
+    """
+    Kullback-Leibler divergence based match. Two modes can be used:
+    if against_negative is True, will return the the log of the ratio of the
```

Comment:
This sentence still make no sense to me, at least not matching what is returned imho.
NB: also typo: 'the the'